### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Sumo99**
Allowed the toggle in the contests page to work properly. I have also updated the css, so that it looks prettier when a toggle happens.

### CODE REVIEW

**Comment by Sumo99** [comment]
Previously, it was not toggling at all

**Comment by rishabhKalakoti** [comment]
Please deploy this branch on github pages so that it can be reviewed easily

**Comment by Sumo99** [comment]
I have deployed it to https://sumo99.github.io/ you can view it there

File: index.html [hunk 1]
```diff
@@ -2,315 +2,523 @@
...
               </div>
-
+              <div class="con">
+                <div class="contest1" onclick="showans('ans2')">
+                  <i class="fas fa-trophy"></i>
+                  Contest 4: (24 Feb 2019)
+                  <i class="fas fa-sort-down" style="float:right"></i>
+                </div>
+                <p id="ans2"><a href="http://hck.re/td11545" target="_blank">Contest 4-Combined for all</a>
+                </p>
```
Comment (rishabhKalakoti) [inline index.html:159]: Do a few updates and then I can merge:
1. Add some padding inside the collapsible body of the contests
2. Remove the rounded borders from the contest accordian

**Review by rishabhKalakoti — CHANGES_REQUESTED** [review]

**Comment by rishabhKalakoti** [comment]
The toggle button seems to work fine on https://codeshows.github.io I don't see any changes in it. Apart from that, you changed the contest page design a little I guess, right? Please try to refer the issues that you are handling by simply writing #<issue number>, it makes the PR more precise.
And next time you deploy, you don't have to deploy the website by creating another repository, sumo99.github.io, you can deploy the existing repo, https://codeshows.github.io by going to the setting and enabling github pages, then it can be viewed at https://sumo99.github.io/codeshows.github.io.
Good going btw, you seem  to have a hang of git already.

File: index.html (hunk 1 above)
Comment (Sumo99) [reply index.html:159]: So you want to the borders to be entirely rectangular? I have this image with the rectangular borders and the padding. 

2019-04-16-134521_1920x1080_scrot
2019-04-16-134532_1920x1080_scrot

Comment (rishabhKalakoti) [reply index.html:159]: This one looks better

**Pull request merged**